docs/architecture/adr-045-admin-module.md

```diff
@@ -0,0 +1,142 @@
+
+# ADR 045: Admin module
+
+## Changelog
+
+- 2021/07/16: Initial draft
+
+## Status
+
+Proposed
+
+## Abstract
+
+This ADR proposes a new module - admin module, that adds simplified governance workflow with authority-based proposal acceptance. A new role is added - administrator. Admins will be capable of:
+- submitting a proposal that will automatically be accepted;
+- add/remove other admins;
+
+## Context
+
+The current proposal workflow is based on a concept of validators (token holders) who vote to accept/reject a proposal. The tallying of those votes is based on the voting power (stake-based voting) of validators and their delegators. This stake-based multi-step approach provides a high level of security necessary for mainnets.
```

Comment:
note, it's not just validators -- any token holder can vote to accept or reject a proposal.